### PR TITLE
Fix broken Mat Synth R&D Values

### LINKED ISSUE #6177: Deconstructing Material Synthesizers gives "3"
* Materials X
* Something X
* 3

Good thing someone tested that

### PR DESCRIPTION
**Pull request by ghost**
I refuse to believe this is only the first time someone notices that, but apparently I had to report it myself

> 3

I did some formatting and span work while in here

Fixes #6177

### CODE REVIEW

File: code/game/objects/items/devices/mat_synth.dm [hunk 1]
```diff
@@ -141,15 +139,15 @@
 /obj/item/device/material_synth/examine(mob/user)
 	..()
 	if(istype(src, /obj/item/device/material_synth/robot))
-		user << "It's been set to draw power from a power cell."
+		user << "<span class='info'>It's been set to draw power from a power cell.</span>"
```
Comment (clusterfack) [inline code/game/objects/items/devices/mat_synth.dm]: This shouldn't be info, its part of the description

**Comment by PJB3005** [comment]
Flatpack fab and reverse engine board have the same issue, could you fix it now you're at it?

**Comment by PJB3005** [comment]
And there seems something weird with the bluespace tomatoes' tech origin values.

**Comment by PJB3005** [comment]
Aaaand the space pod core has `plasma` instead of `plasmatech`.

**Comment by dylanstrategie** [comment]
Will do asap

**Comment by dylanstrategie** [comment]
All were fixed, except for the bluespace tomato. I don't see anything wrong with it

**Comment by PJB3005** [comment]
Yeah I also don't know what's with the tomatoes, but the tech origin list is reporting them seperately.

**Pull request merged**